### ISSUE TITLE
Prevent select-all from selecting all

### ISSUE THREAD

**Issue by tomayac** [romannurik/SlidesCodeHighlighter#22]
This is me friendlily asking @samthor to implement [this solution](https://dev.to/chromiumdev/detecting-select-all-on-the-web-2alo) to make the present product even better.

**Comment by romannurik** [comment]
Is the issue that you sometimes hit <kbd>Cmd+A</kbd> intending to select the highlighted result code, but instead you end up selecting the contents of the page itself? I don't think I've ever tried to run Select All outside the Ace editor :-/

**Comment by tomayac** [comment]
Yes, this exactly! And this is what @samthor's cool [solution](https://dev.to/chromiumdev/detecting-select-all-on-the-web-2alo) fixes.

**Comment by lennertVanSever** [comment]
A solution to this might be a "copy all" button so that you don't have to use Cmd+A  and Cmd+C

**Comment by lennertVanSever** [comment]
Or maybe it's better to automatically copy the code to clipboard when you click anywhere on the right-side panel. Then you also need some kind of toast to notify the user that the code has been copied to his clipboard
